VAMobile/documentation/docs/Engineering/BackEnd/Monitoring/DataDog.md

```diff
@@ -84,4 +84,4 @@ To receive alert messages in slack the Datadog slack tool will need to be added
 
 ## Existing Mobile Dashboards
 
-[Dashboard](https://app.datadoghq.com/dashboard/9nz-cn7-ws6/mobile-api-dashboard) and [Alerts](https://app.datadoghq.com/monitors/manage?q=mobile). If you have issues accessing the dashboard or the VA Datadog site reach out in the shared mobile slack channel
+[Dashboard](https://vagov.ddog-gov.com/apm/services/mobile-app/operations/rack.request/resources?env=eks-prod&panels=qson%3A%28data%3A%28%29%2Cversion%3A%210%29&resources=qson%3A%28data%3A%28visible%3A%21t%2Chits%3A%28selected%3Atotal%29%2Cerrors%3A%28selected%3Atotal%29%2Clatency%3A%28selected%3Ap95%29%2CtopN%3A%215%29%2Cversion%3A%211%29&sort=error-rate%2Cdesc&summary=qson%3A%28data%3A%21f%2Cversion%3A%211%29&view=spans&start=1701356709938&end=1701360309938&paused=false) and [Alerts](https://vagov.ddog-gov.com/monitors/manage?q=mobile&order=desc). If you have issues accessing the dashboard or the VA Datadog site reach out in the shared mobile slack channel
```

Comment:
I updated the Datadog reference to monitors to `vagov.ddog-gov` also because I think all of our monitors have moved over to there from `app.datadoghq.com`. Just wanted to point it out in case I was wrong

Reply:
Everything looks good except the alerts link can just be [this](https://vagov.ddog-gov.com/monitors/90011?view=spans) for now since it's the only one we actively use

Reply:
Ok thanks, updated